From foo@baz Fri Sep 11 14:20:21 PDT 2009
Date: Fri, 11 Sep 2009 14:20:21 -0700
To: Greg KH <greg@kroah.com>
From: Greg Kroah-Hartman <gregkh@suse.de>
Subject: Staging: vt665x: rename the module binary

So it doesn't conflict with a mainline kernel driver
currently under development.

Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 drivers/staging/vt6655/Makefile |    4 ++--
 drivers/staging/vt6656/Makefile |    4 ++--
 2 files changed, 4 insertions(+), 4 deletions(-)

--- a/drivers/staging/vt6655/Makefile
+++ b/drivers/staging/vt6655/Makefile
@@ -2,7 +2,7 @@
 EXTRA_CFLAGS += -DLINUX -D__KERNEL__ -DMODULE -D__NO_VERSION__
 EXTRA_CFLAGS += -DHOSTAP
 
-vt6655-y +=	device_main.o \
+vt6655_stage-y +=	device_main.o \
 	card.o \
 	mac.o \
 	baseband.o \
@@ -35,4 +35,4 @@ vt6655-y +=	device_main.o \
 	vntwifi.o \
 	IEEE11h.o
 
-obj-$(CONFIG_VT6655) +=	vt6655.o
+obj-$(CONFIG_VT6655) +=	vt6655_stage.o
--- a/drivers/staging/vt6656/Makefile
+++ b/drivers/staging/vt6656/Makefile
@@ -2,7 +2,7 @@
 EXTRA_CFLAGS += -DLINUX -D__KERNEL__ -DMODULE  -DEXPORT_SYMTAB -D__NO_VERSION__
 EXTRA_CFLAGS += -DHOSTAP
 
-vt6656-y +=	main_usb.o \
+vt6656_stage-y +=	main_usb.o \
 			card.o  \
 			mac.o   \
 			baseband.o \
@@ -36,4 +36,4 @@ vt6656-y +=	main_usb.o \
 			firmware.o \
 			int.o
 
-obj-$(CONFIG_VT6656) +=	vt6656.o
+obj-$(CONFIG_VT6656) +=	vt6656_stage.o
